ui/theme/theme.go

```diff
@@ -160,6 +170,26 @@ func (m *MyTheme) ListThemeFiles() map[string]string {
 	return result
 }
 
+func (m *MyTheme) createThemeResources() error {
```

Comment:
maybe rename this to `createBuiltinThemeFiles` - for now it's just handling default.toml but in the future if there are other builtin themes, the function name is future-proof